sinon/index.d.ts

```diff
@@ -427,7 +427,7 @@ declare namespace Sinon {
 
     // Utility overridables
     interface SinonStatic {
-        createStubInstance(constructor: any): SinonStub;
+        createStubInstance(constructor: any): any;
```

Comment:
@andy-ms as far as I see, this would work, but how can we also target TS < 2.1?
I assume a lot of projects are still on TS < 2.1 (my project too)